ci-operator/step-registry/cucushift/installer/check/gcp/disktype-control-plane/cucushift-installer-check-gcp-disktype-control-plane-ref.metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+	"path": "cucushift/installer/check/gcp/disktype-control-plane/cucushift-installer-check-gcp-disktype-control-plane-ref.yaml",
```

Comment:
@jianli-wei I did not see the change for my last proposal here ?

Reply:
Sorry, I'd thought you were talking about ipi/conf/gcp/, let me update accordingly, thanks!